Tests/setf-star.lisp

```diff
@@ -76,3 +76,27 @@ RETURN-POLAR is true then return the polar coordinates."
     (signals error
       (setf (my-point-position point :coordinates :angular) (values 1 pi)))))
 
+
+(climi::defgeneric* (setf my-qux-dufbar) (arg1 arg2
+                                               &optional
+                                               kaboom1
+                                               kaboom2
+                                               &key
+                                               ((:duf uno))
+                                               ((:qux duo))))
+
+(climi::defmethod* (setf my-qux-dufbar) (arg (point my-point)
+                                             &optional
+                                             kaboom1
+                                             kaboom2
+                                             &key
+                                             ((:duf one))
+                                             ((:qux two)))
+  (setf (my-point-x point) arg)
+  (setf (my-point-y point) (+ one two kaboom1 kaboom2)))
+
+(test my-qux-dufbar-test
+ (let ((point (make-instance 'my-point :x 0 :y 0)))
+   (setf (my-qux-dufbar point 1 2 :duf 3 :qux 4) 5)
+   (is (and (= 5 (my-point-x point))
+            (= 10 (my-point-y point))))))
```

Comment:
Seems to need one additional column of indentation.

Reply:
fixed